bin/run

```diff
@@ -28,16 +28,15 @@ wait_for() {
 
 # Only wait for backend services in development
 # http://stackoverflow.com/a/13864829
-[ ! -z ${DEVELOPMENT+check} ] && wait_for db 5432 && wait_for redis 6379
+# [ ! -z ${DEVELOPMENT+check} ] && wait_for db 5432 && wait_for redis 6379
 
 case $1 in
   web)
-    newrelic-admin run-python manage.py migrate --noinput
-    exec newrelic-admin run-program gunicorn taar_api.wsgi:application -b 0.0.0.0:${PORT} --workers 4 --access-logfile -
+    #exec newrelic-admin run-program gunicorn taar_api.app:app -b 0.0.0.0:${PORT} --workers 4 --access-logfile -
```

Comment:
lets take all these commented out lines before we merge. also in dockerfile.

Reply:
Fixed now.  I've also parameterized the port and host for the web-dev target by adding an extra wsgi wrapper, so bin/run is very minimally altered now.